src/main/kotlin/racingcar/strategy/NumberGeneratorStrategy.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar.strategy
+
+interface NumberGeneratorStrategy {
+    fun generateNumber(): Int
+
+    fun isMovable(): Boolean
```

Comment:
```suggestion
interface MovableStrategy {
    fun isMovable(): Boolean
```
isMovable 함수만 추상화해보면 어떨까요?
Number관련된 책임은 구현체에서 구현하면 될거같아요!